### PR TITLE
Exercise 2 - Use Trello API and add ability to mark items as done

### PR DESCRIPTION
**Pull request by SChandarana**

### CODE REVIEW

File: todo_app/data/trello_items.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import os
+import requests
+from requests.api import request
```
Comment (JoshAdamPowell) [inline todo_app/data/trello_items.py:3]: Super minor, but I dont think this import is used?

File: todo_app/data/trello_items.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    response = requests.request(
+        'GET',
+        url,
+        params={
+            'key': key,
+            'token': token,
+            'cards': 'open',
+            'fields': 'name'
+        }
+    )
```
Comment (JoshAdamPowell) [inline todo_app/data/trello_items.py:50]: again very minor but this block is repeated just above

Comment (JoshAdamPowell) [reply todo_app/data/trello_items.py:50]: (I accept that the URL varies but I'm sure you can work a way to get past that :P )

**Review by JoshAdamPowell — APPROVED** [review]
Perfect exercise, I've left a couple of very minor optional improvements. Happy for this work to be merged anyway though 😄

File: todo_app/data/trello_items.py (hunk 1 above)
Comment (SChandarana) [reply todo_app/data/trello_items.py:3]: Yep, I'll sort these out as part of the other exercise :D

**Pull request merged**